### PR TITLE
Control panel

### LINKED ISSUE #36: Replace "Boolean" with "boolean"
https://github.com/2729StormRobotics/StormRobotics2020/blob/58aca8f42bfbd73810ecd9a0b202380c8f17564a/src/main/java/frc/robot/subsystems/ControlPanel.java#L95

https://lgtm.com/projects/g/2729StormRobotics/StormRobotics2020/snapshot/ea1de0de0daf442b791ffca645d4f4d2118c3511/files/src/main/java/frc/robot/subsystems/ControlPanel.java?sort=name&dir=ASC&mode=heatmap#xec0a28ab38de6df1:1

Explanation: https://lgtm.com/rules/1813633265/

### LINKED ISSUE #39: Use .equals("Red") instead of == "Red"
https://github.com/2729StormRobotics/StormRobotics2020/blob/58aca8f42bfbd73810ecd9a0b202380c8f17564a/src/main/java/frc/robot/subsystems/ControlPanel.java#L83-L91

Explanation: https://sonarcloud.io/organizations/storm-robotics-2729-github/rules?open=java%3AS4973&rule_key=java%3AS4973

### PR DESCRIPTION
**Pull request by hrose967**
Fixes #36 and #39

### CODE REVIEW

**Review by PKuppili — DISMISSED** [review]
I didn't see any constructor. You might want to add that if you need to add anything in it.

File: src/main/java/frc/robot/subsystems/ControlPanel.java [hunk 1]
```diff
@@ -80,19 +73,69 @@ public int colorCount(){
...
+      break;
+  }
+} else {
+  //Code for no data received yet
+}
+
+}
+public void findBlue(){
+  startMotors(.18);
+  if (color == "Red")
```
Comment (PKuppili) [inline src/main/java/frc/robot/subsystems/ControlPanel.java]: Should .equals()

**Review by zbross — APPROVED** [review]
Approved

**Review by zbross — APPROVED** [review]

**Review by zbross — APPROVED** [review]
Approved

**Pull request merged**